cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -912,7 +914,44 @@ def test_can_import_annotations_for_image_with_dots_in_filename(self):
             self._generate_annotations(task, format_name)
 
             with self.subTest(format=format_name):
-                if not f.ENABLED:
+                if not f.ENABLED or format_name in {'Sly Point Cloud Format 1.0',
+                    'Kitti Raw Format 1.0'}:
                     self.skipTest("Format is disabled")
 
                 self._test_can_import_annotations(task, format_name)
+
+    def test_can_import_mots_annotations_with_splited_masks(self):
+        #https://github.com/openvinotoolkit/cvat/issues/3360
+        from datumaro.components.dataset import Dataset, DatasetItem
+        from datumaro.components.extractor import Mask
+        from numpy import array
```

Comment:
What is the point of such import?

Reply:
These classes are only needed in this test, so I decided to import them inside the function.

Reply:
It's not a significant reason for having them here.

Also, numpy is typically imported as `import numpy as np`